src/storyMap/components/StoryMapForm/EditableMedia.js

```diff
@@ -441,6 +441,80 @@ const EditableAudio = props => {
   );
 };
 
+const EditableVideo = props => {
+  const { t } = useTranslation();
+  const [id, setId] = useState(0);
+  const { getMediaFile } = useStoryMapConfigContext();
+  const { video, onUpdate, onDelete, processing } = props;
+
+  useEffect(() => {
+    setId(uuidv4());
+  }, [video]);
+
+  const videoSrc = useMemo(() => {
+    if (video.signedUrl) {
+      return video.signedUrl;
+    }
+    if (video.contentId) {
+      return getMediaFile(video.contentId);
+    }
+    return null;
+  }, [video, getMediaFile]);
+
+  return (
+    <Stack>
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <video key={id} style={{ width: '100%' }} controls>
+        <source src={videoSrc} type={video.type} />
+        {t('storyMap.form_media_audio_not_supported')}
+      </video>
+      <Stack
+        justifyContent="center"
+        alignItems="center"
+        direction="row"
+        sx={{
+          color: 'white',
+          background: 'rgba(0,0,0,0.5)',
+          bottom: 0,
+          width: '100%',
+          pt: 2,
+          pb: 2,
+        }}
+        spacing={1}
+      >
+        <Button
+          variant="outlined"
+          onClick={onUpdate}
+          sx={({ palette }) => ({
+            backgroundColor: 'white',
+            '&:hover': {
+              backgroundColor: palette.blue.background,
+            },
+          })}
+        >
+          {t('storyMap.form_media_update_label')}
+        </Button>
+        <ConfirmButton
+          onConfirm={onDelete}
+          loading={processing}
+          variant="text"
+          buttonProps={{
+            title: t('storyMap.form_media_delete_label'),
+            sx: {
+              minWidth: 'auto',
+            },
+          }}
+          confirmTitle={t('storyMap.form_media_video_delete_confirm_title')}
```

Comment:
Question: Are these string sin the en_us file?

Reply:
Ya, it was part of another ticket that was already merged in main